### PR TITLE
vm: FIX - Add dropdown select IP for network type is Shared

### LINKED ISSUE #759: [BUG]  VM Instance, Change IP , Do not have DropDown List to Select
**Describe the bug**
Primate Interface, VM instance , Change IP Address,  only Manual key in IP address to change. It shall be list down IP to select to change , Old Interface has this. 

**To Reproduce**
1. Log in as a User or an Admin
2. Go to 'Instance, Select the VM'
3. Click on 'NIC' and Click '-'
4. Click Change IP icon, and it will pop up new window for key in IP
5. The IP to key in is manual keyin, it shall be select from drop down list select ip avaible to change

**Expected behavior**
It shall list out the available IP to change, instead of Admin Key in whatever they want .
In old interface, this is dropdown selection.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Mac OS
 - Browser Chrome
 

**Additional context**

### PR DESCRIPTION
**Pull request by utchoang**
Fixes #759 
@rhtyd cc @svenvogel 
Please review it. Thanks.
image
image

### CODE REVIEW

**Comment by rhtyd** [comment]
@blueorangutan package

File: src/views/compute/InstanceTab.vue [hunk 1]
```diff
@@ -354,6 +370,45 @@ export default {
         this.secondaryIPs = response.listnicsresponse.nic[0].secondaryip
       })
     },
+    fetchGuestNetwork (networkid) {
```
Comment (davidjumani) [inline src/views/compute/InstanceTab.vue:373]: @utchoang Is this method used anywhere ?

Comment (utchoang) [reply src/views/compute/InstanceTab.vue:373]: @davidjumani No. This is a function I testing. I'll remove it. Thanks.

**Comment by davidjumani** [comment]
@blueorangutan package

**Review by davidjumani — APPROVED** [review]
LGTM! Thanks @utchoang

**Pull request merged**